### DISCUSSION TITLE
New to this project

### DISCUSSION THREAD

**Discussion by ShaikAslam224** [vesoft-inc/nebula#4774] [General]
hey hello everyone! iam Aslam .
iam new to this opensource & projects. Please help anybody to participle in this project. I know python!

**Comment by wey-gu** [comment]
Hey @ShaikAslam224 

Welcome to the NebulaGraph community and OSS world :).

Glad to have you here!

If you prefer to contribute on Python, you could check issues from those repos to see if you would like to contribute things there.

- https://github.com/vesoft-inc/nebula-python the NebulaGraph Python client
- https://github.com/nebula-contrib/nebula-carina NebulaGraph Python ORM
- https://github.com/vesoft-inc/nebula-spark-connector/ NebulaGraph Spark connector, supports PySpark

Also, feel free to learn NebulaGraph from https://nebula-graph.io/ and all contents there are backed here: https://github.com/vesoft-inc/nebula-docs , where you could make contributions to, too.

Let me know when you need more helps on anything in NebulaGraph :)
